helm_drydock/config.py

```diff
@@ -21,21 +21,13 @@
 
```

Comment:
No need to prefix this with `helm_` as it may confuse that its a helm-based chart.

It is simply `drydock` - later there will be a `drydock` helm chart, which will likely live in `aic-helm`, as well as a drydock container living in `dockerfiles` - the same pattern as Anthony has followed for ShipYard.